scripts/environment_check.sh

```diff
@@ -250,6 +250,18 @@ check_hostname_uniqueness() {
   info "All nodes have unique hostnames."
 }
 
+check_default_priority_class() {
+  hasDefault=$(kubectl get priorityclass -o jsonpath='{.items[*].globalDefault}' | grep true)
+
+  if [ ! -z $hasDefault ]; then
+      # See https://longhorn.io/kb/troubleshooting-instance-manager-pods-are-restarted-every-hour/#reason
+      warn "Cluster has a global default priority class."
```

Comment:
Can we link the recommended action into the log for user to see it? 
```suggestion
      warn "Cluster has a global default priority class. See potential issue and solution at https://longhorn.io/kb/troubleshooting-instance-manager-pods-are-restarted-every-hour/#reason"
```

Reply:
Done.